src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitsStorage = new HashMap<>();
+
+    public Map<String, Integer> getFruitsStorage() {
+        return fruitsStorage;
```

Comment:
Returning a direct reference to the `fruitsStorage` map can lead to unintended modifications from outside the class. Consider returning an unmodifiable view or a copy of the map to prevent external modifications.